lucien-ios/Add New Comic Book/AddComicViewController.swift

```diff
@@ -0,0 +1,208 @@
+                         "Children’s",
+                         "Comedy",
+                         "Crime",
+                         "Drama",
+                         "Fantasy",
+                         "Graphic Novels",
+                         "Historical",
+                         "Horror",
+                         "LGTBQ",
+                         "Literature",
+                         "Manga",
+                         "Mature",
+                         "Movies & TV",
+                         "Music",
+                         "Mystery",
+                         "Non-Fiction",
+                         "Original Series",
+                         "Political",
+                         "Post-Apocalyptic",
+                         "Religious",
+                         "Romance",
+                         "School Life",
+                         "Science Fiction",
+                         "Slice of Life",
+                         "Superhero",
+                         "Supernatural/Occult",
+                         "Suspense",
+                         "Western"
+    ]
+
+    // MARK: - Private IBOutlets
+
+    @IBOutlet private weak var coverPhotoLabel: UILabel!
+    @IBOutlet private weak var coverPhotoButton: UIButton!
+    @IBOutlet private weak var comicTitleLabel: UILabel!
+    @IBOutlet private weak var comicTitleTextField: UITextField!
+    @IBOutlet private weak var storyTitleLabel: UILabel!
+    @IBOutlet private weak var storyTitleTextField: UITextField!
+    @IBOutlet private weak var volumeLabel: UILabel!
+    @IBOutlet private weak var volumeTextField: UITextField!
+    @IBOutlet private weak var issueLabel: UILabel!
+    @IBOutlet private weak var issueTextField: UITextField!
+    @IBOutlet private weak var publisherLabel: UILabel!
+    @IBOutlet private weak var publisherTextField: UITextField!
+    @IBOutlet private weak var releaseDateLabel: UILabel!
+    @IBOutlet private weak var genreLabel: UILabel!
+    @IBOutlet private weak var conditionLabel: UILabel!
+    @IBOutlet private weak var selectADateButton: UIButton!
+    @IBOutlet private weak var selectAGenreButton: UIButton!
+    @IBOutlet private weak var selectAConditionButton: UIButton!
+    @IBOutlet private weak var datePicker: UIDatePicker!
+    @IBOutlet private weak var conditionPicker: UIPickerView!
+    @IBOutlet private weak var genrePicker: UIPickerView!
+    @IBOutlet private weak var scrollView: UIScrollView!
+    @IBOutlet private weak var releaseDateBottomLine: UIView!
+    @IBOutlet private weak var genreBottomLine: UIView!
+    @IBOutlet private weak var conditionBottomLine: UIView!
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        configureViewController()
+    }
+
+    // MARK: - Private Instance Methods
+
+    private func configureViewController() {
+        // Cover Photo Cell
+        coverPhotoLabel.font = UIFont(name: LucienTheme.Fonts.Muli.light.rawValue, size: 12)
```

Comment:
We'll discuss this further, but usually the idea with a Theme file is that it provides you the font itself, not just the name, so you don't have to create it on the fly

Reply:
I couldn't make an enum with UIFonts, but I think an alternative is to have a struct that provides the UIFonts directly. Would this be a better idea?